docs/pages/setup/admin/labels.mdx

```diff
@@ -1,66 +1,274 @@
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+On your Node, record the address of your Auth Service:
+
+```code
+$ ADDR=tele.example.com:3025
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+On your Node, record the address of your Teleport Cloud tenant:
+
+```code
+$ ADDR=mytenant.teleport.sh:443
+```
+
+</ScopedBlock>
+
+Start Teleport:
+
+```code
+$ sudo teleport start \
+  --auth-server=${ADDR?} \
+  --roles=node \
+  --labels environment=dev \
+  --token=${TOKEN?}
+```
+
+
+</TabItem>
+<TabItem label="Config File">
+
+You can apply labels to your Node by editing the Node's configuration file,
+`/etc/teleport.yaml`, to include the following content:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
 
 ```yaml
+version: v2
+teleport:
+  auth_servers:
+    - tele.example.com:3025
+
+auth_service:
+  enabled: false
+
+proxy_service:
+  enabled: false
+
 ssh_service:
-  enabled: "yes"
-  # ...
+  enabled: true
   # Static labels are simple key/value pairs:
   labels:
-    environment: test
-app_service:
-  # ..
+    environment: dev
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```yaml
+version: v2
+teleport:
+  auth_servers:
+    - mytenant.teleport.sh:443
+
+auth_service:
+  enabled: false
+
+proxy_service:
+  enabled: false
+
+ssh_service:
+  enabled: true
+  # Static labels are simple key/value pairs:
   labels:
-    environment: test
+    environment: dev
+```
+
+</ScopedBlock>
+
+Next, start Teleport with the join token you created earlier:
+
+```code
+$ sudo teleport start --token=${TOKEN?}
 ```
 
-To configure dynamic labels via a configuration file, define a `commands` array
-as shown below:
+</TabItem>
+</Tabs>
+
+Verify that you have added the label by running the following on your local
+machine. Your Teleport user must be authorized to access the Node.
+
+```code
+$ tsh ls --search environment=dev
+Node Name         Address    Labels          
+----------------- ---------- --------------- 
+ip-192-0-2-0 ⟵ Tunnel   environment=dev 
+```
+
+## Step 3/3. Apply dynamic labels via commands
+
+Before following this step, stop Teleport on your Node.
+
+As with static labels, you can apply dynamic labels either when starting
+Teleport on the command line or by editing the Teleport configuration file.
+
+<Tabs>
+<TabItem label="Command Line">
+
+To define labels as command line arguments, use the `--labels` flag as shown
+below. This method works well for simple commands.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+On your Node, record the address of your Auth Service:
+
+```code
+$ ADDR=tele.example.com:3025
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+On your Node, record the address of your Teleport Cloud tenant:
+
+```code
+$ ADDR=mytenant.teleport.sh:443
+```
+
+</ScopedBlock>
+
+Start Teleport:
+
+```code
+$ sudo teleport start \
+  --auth-server=${ADDR?}\
+  --roles=node \
+  --token=${TOKEN?} \
+  --labels uptime=[1m:"uptime -p"],kernel=[1h:"uname -r"]
```

Comment:
We are trying to get away from recommending long `teleport start` commands with lots of complicated flags in favor of simpler `teleport start -c teleport.yaml` start commands - thus pushing the configuration into a file.

Given this, and the fact that this syntax for specifying labels on the command line is pretty complicated, I would prefer we just recommend the config file approach here too.